Sources/SwiftParser/Modifiers.swift

```diff
@@ -201,7 +201,7 @@ extension Parser {
     } else {
       unexpectedBeforeDetail = nil
       detail = RawTokenSyntax(
-        missing: .keyword(.set),
+        missing: .keyword,
         text: "set",
```

Comment:
`Keyword.set.defaultText`? (and the try/etc ones)

Or alternatively have a `RawTokenSyntax(mising:arena:)` where `missing` is a `Keyword` that does automatically.

Reply:
Good idea. I added `missingToken(Keyword)` as a method on `Parser`.